loader/base_loader.py

```diff
@@ -368,6 +374,12 @@ def _upload_tagged_cloud_file_to_dss_by_copy(self, source_bucket: str,
         # and we need this format update when doing load bundle
         file_version = response.json().get('version', "blank")
 
+        # from dss swagger docs:
+        #   > 200 Returned when the file is already present and is identical to the file being uploaded.
+        if response.status_code == requests.codes.ok:
+            already_present = True
+        else:
+            already_present = False
         if response.status_code in (requests.codes.ok, requests.codes.created):
```

Comment:
I think `requests.codes.ok` doesn't need to be in this set, correct?
If we get an ok (200) that means the file is already present and identical, so no sync copy will be performed, so we don't need to log that and wait for the `head_file` to indicate that the file appeared.

Reply:
It does! I actually made this mistake originally thinking just as you did.  If you take it out then you get stuck in the else block bellow and time out. But you did make me realize this could be shortened to
```
already_present = response.status_code == requests.codes.ok
```

Reply:
It doesn't! 😃 ... but additional code changes are needed in this case, which is what I had in mind but didn't explicitly describe.

There four specific return codes we want to handle here, each with a different log message:
- requests.codes.ok -> file already exists
- requests.codes.created -> Sync copy
- requests.codes.accepted -> Async copy 
- requests.codes.conflict -> File with same UUID+version exists but the new file is different from the existing file, so the new file is not loaded.

Let's talk about this before you start making changes to ensure the cases are clear.